config/locales/en.yml

```diff
@@ -2900,6 +2915,23 @@ en:
       showing_page: "Page %{page}"
       next: "Next »"
       previous: "« Previous"
+  user_mutes:
+    index:
+      you_have_muted_n_users:
+        zero: "You haven't muted any Users 🎉"
+        one: "You have muted %{count} User"
+        other: "You have muted %{count} Users"
```

Comment:
Why are we capitalising User/Users here?

Reply:
Because all actions containing the keyword `User` on `Users#show` are currently capitalised.

![SCR-20231023-lnud](https://github.com/openstreetmap/openstreetmap-website/assets/28648/769b4874-e74c-4896-8dbc-355ef5c2a8f5)

Reply:
ah. I understand that here in this context "Users" should not be capitalised since it's not referencing a language subject. I'll adjust the translation.